test/XRP/fakes/fake-grpc-error.ts

```diff
@@ -9,12 +7,9 @@ export default class FakeGRPCError extends Error {
    * Construct a new FakeServiceError.
    *
    * @param message The text details of the error.
-   * @param code The grpc.status code.
+   * @param code The gRPC status code.
    */
-  constructor(
-    public readonly message: string,
-    public readonly code: grpc.status,
-  ) {
+  constructor(public readonly message: string, public readonly code: number) {
```

Comment:
typed as `number` because `grpc-web.StatusCode` is a `namespace`, which doesn't generate a type